lms/templates/register.html

```diff
@@ -282,6 +282,19 @@ <h2 class="sr">${_("Additional Personal Information")}</h2>
             </div>
           </li>
           % endif
+          % if settings.REGISTRATION_EXTRA_FIELDS['dropdown'] != 'hidden':
```

Comment:
Please change this so it doesn't cause a 500 error when the configuration doesn't contain `'dropdown'` since nobody's existing deployments have that key in their config.